.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.5.0",
-  "packages/api-explorer": "0.9.6",
-  "packages/code-editor": "0.1.0",
-  "packages/extension-api-explorer": "21.4.5",
-  "packages/extension-sdk": "21.4.5",
-  "packages/extension-sdk-react": "21.4.5",
-  "packages/hackathon": "21.4.5",
-  "packages/run-it": "0.9.6",
-  "packages/sdk": "21.4.3",
-  "packages/sdk-codegen": "21.0.15",
-  "packages/sdk-codegen-scripts": "21.0.15",
+  ".": "1.6.0",
+  "packages/api-explorer": "0.9.7",
+  "packages/code-editor": "0.1.1",
+  "packages/extension-api-explorer": "21.4.6",
+  "packages/extension-sdk": "21.4.6",
+  "packages/extension-sdk-react": "21.4.6",
+  "packages/hackathon": "21.4.6",
+  "packages/run-it": "0.9.7",
+  "packages/sdk": "21.5.0",
```

Comment:
@jkaster do you want to take a stab at modifying `release-please-config.json` so that we get the correct versions on this PR? Happy to meet on it. Or I can just do it but hoping to spread the experience :-)

Reply:
Does this manifest need to change as well? I'd like to pair up to make the changes so we can review the process.

Reply:
to answer your question: the manifest is supposed to only be written to by release-please - however, as part of my manual work-around for the bug with extension-{sdk,sdk-react,api-explorer} version bumping I did manually update the manifest (along with the package.json files and the changelogs)